server/go/pkg/server/options.go

```diff
@@ -0,0 +1,37 @@
+package server
+
+import (
+	"errors"
+	"strings"
+	"time"
+)
+
+//nolint:lll,staticcheck
+type Options struct {
+	ServiceName      string        `long:"service" required:"yes" description:"The name of this service" env:"ASHERAH_SERVICE_NAME"`
```

Comment:
have been calling it ServiceId and ProductId at every place it seems. Better to decide on a consistent naming approach. Same with all the other options, I think we can decide on a consistent naming schema for all the available options for Java and Go servers

Reply:
Agreed. We need to introduce config consistency between implementations across the board... a unified configuration file, consistent flags, etc. My vote is to leave this as-is for now and revisit when we take up that effort.